### ISSUE TITLE
Yeah you guys wanna update this...

### ISSUE THREAD

**Issue by dustinrouillard** [briantanner/lavalink.js#5]
If you could update this to V2 of lavalink that would be great. kthx.

**Comment by Bacon1661** [comment]
Yup. I already contacted Jacz about this and he says he's going to make his own client. He didn't give a date but hopefully, it'll be done soon.

**Comment by iDerp** [comment]
Awesome

**Comment by ohlookitsdonut** [comment]
Is this one getting updated to V2, orrrr?

**Comment by Bacon1661** [comment]
@ohlookitsdonut I don't think so, I think this one was made to replace it: https://github.com/MrJacz/discord.js-lavalink
